src/groups/Members/EmailSelect.tsx

```diff
@@ -1,80 +1,67 @@
-import { CreatableSelect, useUniqueId } from '@terra-ui-packages/components';
+import { useUniqueId } from '@terra-ui-packages/components';
 import _ from 'lodash/fp';
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
+import { TextInput } from 'src/components/input';
 import { FormLabel } from 'src/libs/forms';
 
 interface EmailSelectProps {
   label?: string;
   placeholder?: string;
-  isMulti?: boolean;
-  isClearable?: boolean;
-  isSearchable?: boolean;
-  options: string[];
-  emails: string[];
   setEmails: (values: string[]) => void;
+  emails: string[];
 }
 
 export const EmailSelect: React.FC<EmailSelectProps> = ({
   label = 'User emails',
-  placeholder = 'Type or select user emails',
-  isMulti = true,
-  isClearable = true,
-  isSearchable = true,
-  options,
-  emails,
+  placeholder = 'Type user emails separated by commas',
```

Comment:
This seemed a more appropriate placeholder but I'm open to suggestions on better wording